man/man1/git-secret-killperson.1.ronn

```diff
@@ -3,7 +3,7 @@ git-secret-killperson - deletes key identified by an email from the inner keyrin
 
 ## SYNOPSIS
 
-    git secret killperson <emails>...
+    git secret killperson [-f] <emails>...
```

Comment:
Let's make this use `-F` instead of `-f`, then it's more consistent with other git-secret options.